app/models/question.rb

```diff
@@ -17,4 +17,16 @@ class Question < ApplicationRecord
   has_many :question_answers
 
   enum answer_type: [:open, :single, :multiple]
+
+  def is_open?
```

Comment:
In Ruby we only use open?, single? and multiple? for methods that return a boolean.